projects/gateway2/helm/gloo-gateway/templates/gateway/proxy-deployment.yaml

```diff
@@ -401,9 +401,10 @@ metadata:
   labels:
     {{- include "gloo-gateway.gateway.constLabels" . | nindent 4 }}
     {{- include "gloo-gateway.gateway.labels" . | nindent 4 }}
-{{- if $gateway.aiExtension.stats }}
+{{- if or $gateway.aiExtension.stats $gateway.aiExtension.tracing }}
 data:
   stats.json: {{ $gateway.aiExtension.stats }}
+  tracing.json: {{ $gateway.aiExtension.tracing }}
```

Comment:
Because of the `or` stats or tracing can be null? The `or` is for if we should have the `data:` section and inside `data:`, still need to check each before setting them?

Reply:
Good point, I can add another `if`